lint/util.py

```diff
@@ -610,3 +610,10 @@ def load_json(*segments, from_sl_dir=False):
     base_path = "Packages/SublimeLinter/" if from_sl_dir else ""
     full_path = base_path + "/".join(segments)
     return sublime.decode_value(sublime.load_resource(full_path))
+
+
+def get_current_pos(view):
+    try:
+        return view.rowcol(view.sel()[0].begin())
+    except (AttributeError, IndexError):
+        return -1, -1
```

Comment:
I believe this should return a `sublime.Region` - or the above should return a tuple.

Considering that the default for `current_pos` in the `State` dicts is a tuple, probably the latter.

Reply:
Good spot. It is a potential issue.
This pattern has not been introduced by this PR and it works nonetheless.
So, let's just fix this in a consecutive PR.